app/models/rent.rb

```diff
@@ -252,24 +226,24 @@ def p_email
   def as_json(options = {})
     if service
       {
-          :id => self.id,
-          :title => "Service",
-          :start => self.d_from.rfc822,
-          :end => self.d_til.rfc822,
-          :status => self.comment,
-          :backgroundColor => "black",
-          :textColor => "white"
+         id: id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.